Dockerfile

```diff
@@ -22,14 +20,15 @@ RUN apt-get update; \
   apt-get autoremove && \
   apt-get clean
 
-COPY --chown=app:app Gemfile /home/app/Gemfile
+WORKDIR /home/app
+
+COPY --chown=app:app Gemfile ./Gemfile
 
-RUN cd /home/app && \
-    mkdir /home/app/tmp && \
-    bundle install --gemfile /home/app/Gemfile
+RUN bundle install --gemfile Gemfile
 
 # Install the app (including Gemfile)
-COPY --chown=app:app . /home/app
+COPY --chown=app:app . .
 
-CMD bundle exec puma -t 1:5 -w 1 --preload -p 80
+USER app
 
+CMD bundle exec puma -t 1:5 -w 1 --preload -p 3000
```

Comment:
Regarding this one, we should put something that makes sense directly in startback (and release 0.9.2 if needed) and stop putting the command explicitely everywhere.

Right @llambeau?

Reply:
Oh I think startback is doing it already here https://github.com/enspirit/startback/blob/ddba53c18d82584d972d898ac32920090ba8c6ee/Dockerfile.api#L12